doc/nrf/releases/release-notes-latest.rst

```diff
@@ -140,6 +225,14 @@ The following list summarizes the most important changes inherited from upstream
     * Added support for sub-word lengths of read and write transfers in the nRF QSPI NOR flash driver (nrf_qspi_nor).
     * Improved the handling of erase operations in the nRF QSPI NOR flash driver (nrf_qspi_nor), the AT45 family flash driver (spi_flash_at45), and the SPI NOR flash driver (spi_nor).
       Now the operation is not started if it cannot be completed successfully.
+    * Established the unrestricted alignment of flash reads for all drivers.
+    * Enhanced the nRF QSPI NOR flash driver (nrf_qspi_nor) so that it supports unaligned read offset, read length, and buffer offset.
+    * Added SFDP support in the SPI NOR flash driver (spi_nor).
+    * Fixed a regression in the nRF flash driver (soc_flash_nrf) when using the :option:`CONFIG_BT_CTLR_LOW_LAT` option.
+
+  * EEPROM:
+
+    * Fixed at2x cs gpio flags extraction from DT.
```

Comment:
I don't understand what we're trying to say here.

Reply:
Rephrased and moved above.